tests/cypress/support/stateSubmissionFormCommands.ts

```diff
@@ -258,12 +256,6 @@ Cypress.Commands.add(
                 'exist'
             ) 
         } else if (buttonKey === 'CONTINUE'){
-            if (waitForLoad){
-                cy.wait('@updateDraftSubmissionMutation')
-                cy.wait('@fetchHealthPlanPackageQuery')
-            }
```

Comment:
just FYI this was only added to address the flakes that happened after OTEL. Let's keep an eye on Cypfess